src/Transports/Seatrac/Task.cpp

```diff
@@ -627,11 +622,6 @@ namespace Transports
               {
                 sendProtectedCommand(commandCreateSeatrac(CID_DAT_SEND, m_data_beacon));
               }
-              else
-              {
-                // Data communication done
-                clearTicket(IMC::UamTxStatus::UTS_DONE);
```

Comment:
@JoaoTeixiera For future review: I have tested this line while I was testing the usbl task (Both manta-12 and caravela) and I didn't get any core dump into log folder.